src/assets/sass/_variables.scss

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap');
+
+// COLORS
+$primary-color: #B4DDC0;
+$secondary-color: #005546;
+$tertiary-color: #F8F8F8;
+$quaternary-color: #2F2D53;
+$quinary-color: #AFCAC5;
+$senary-color: #2C2C2C;
+
+
+$white-color: white;
+$black-color: black;
+
+// FONTS
+
```

Comment:
Let's add some sizes for fonts.

![image](https://user-images.githubusercontent.com/55165280/175987717-24f35760-e71d-4c5e-be68-1f59db16ec05.png)